### DISCUSSION TITLE
[UserStoriesToRefine] #1 As <user> I want a website that clearly communicates United by Chocolate, a non-profit that is here to empower farmer communities through translated information for the cacao and chocolate industry

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#20] [General]
### As <user> I want a website that clearly communicates United by Chocolate, a non-profit that is here to empower farmer communities through translated information for the cacao and chocolate industry

**Comment by UnitedByChocolate** [comment]
## As a [USER NAME] I can [ACHIEVE SOMETHING], so that [VALUE STATEMENT]
## Level 2- As a cacao producer I want to locate videos, so that I can be better informed about the chocolate industry.
### Level 3 - As a cacao producer I want to be able be click logos that will locate videos, so that I can be better informed about the chocolate industry.
### Level 3 - As a cacao producer I want be able to type keywords into a search bar, so that I get a list of matching videos
### Level 3 - As a cacao producer I want be able to see playlists, so that I filter and focus on my topic/ category of preference 
### Level 3 - As a cacao producer I want be able to see suggested videos, so that I can find related, helpful information that I didn't know existed.

**Reply by emecas** [reply]
Regarding to the first of the level 3 stories:

> ### Level 3 - As a cacao producer I want to be able be click logos that will locate videos, so that I can be better informed about the chocolate industry.

Even, I am not too convinced yet about the wording of  the _SO THAT PART_ , mainly because I think it's still too fuzzy and high level, I have prepared a list of possible acceptance criteria:

**Acceptance Criteria:**

1. One list/bar of logos (icons) with different categories is available to click
2. When the user click on one of the logos, a list of filtered videos related to the particular logo-category is presented
3. the presented logos/icons are consistent with the categories them represent
4. All allocated videos once a logo is clicked are consistent with the concept of the logo clicked 
5. The list of allocated is ordered according to percentage of relevance

**Reply by nhrafiq** [reply]
Here is what I recall from the acceptance criteria Esther and I came up with at the end of the last meeting:

> Level 3 - As a cacao producer I want be able to type keywords into a search bar, so that I get a list of matching videos
* Keywords searches on video descriptions, titles, and tags for corresponding matches 
* The search results appear as a list with video thumbnail and video title / tags (focusing on highlighting each video's visual elements for easier searching)

> Level 3 - As a cacao producer I want be able to see playlists, so that I filter and focus on my topic/ category of preference
* Playlists could have image banners associated with them so their meaning can be understood from the visual elements. 
* Users can add videos to playlists in order to create community-made playlists which might better correspond to where users believe certain videos should be categorized. 

> Level 3 - As a cacao producer I want be able to see suggested videos, so that I can find related, helpful information that I didn't know existed.
* Suggestions are made based off of a user's recent searches and video-watching history, as well as what users who made similar searches or watched the same videos have also watched 
* Users can give feedback as to whether a suggested video was in fact relevant to them or not in order to refine the suggestion process.

**Reply by UnitedByChocolate** [reply]
Wow, thank you so much Noor!
